### PR TITLE
Add author names and description

### PR DESCRIPTION
**Pull request by shunkosa**
Hello @Confirm4Crit! Thank you for adding my repo to this curation. I added
* Author name with the link
* Update name and description
* Remove the POC repo and the lookup repo (leaving the more popular one)

Hope you'll review this :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,49 +12,49 @@ Join the [SFXD Discord Community](https://discordapp.com/invite/JG4Bc4q)!
...
 
-- [LWC Listview](https://github.com/shliachtx/lwc-listview)
+- [User Interactions with Platform Events and LWC](https://github.com/lucianostraga/userInteractions-platformEvents-lwc) (Luciano Straga, [@lucianostraga](https://github.com/lucianostraga))
 
-  A dynamic datatable for use in flexipages or standalone 
+  An user interaction tracker with Lightning Web Components and Platform Events for Lightning Experience and Communities
+
+- [LWC Listview](https://github.com/shliachtx/lwc-listview) (Menachem Lazaroff, [@shliachtx](https://github.com/shliachtx))
 
-- [LWC Lookup Datatable](https://github.com/SFXD/LWC-Lookup-Datatable)
```
Comment (Confirm4Crit) [inline README.md]: Looks like we lost this in the shuffle, was that intentional?

Comment (shunkosa) [reply README.md]: Sorry, recovered!

Comment (Confirm4Crit) [reply README.md]: All good!

File: README.md [hunk 2]
```diff
@@ -12,49 +12,49 @@ Join the [SFXD Discord Community](https://discordapp.com/invite/JG4Bc4q)!
...
+  A dynamic datatable for use in flexipages or standalone 
 
-- [LWC Lookup](https://github.com/jlyon87/lwc-lookup)
```
[the remark this answers is not in the document]
Comment (Confirm4Crit) [reply README.md]: @gdoenlen Didn't catch that, readded.

**Pull request merged**